### ISSUE TITLE
v1/groups/{phone#] API endpoint returning Go struct error

### ISSUE THREAD

**Issue by kocsenc** [bbernhard/signal-cli-rest-api#187]
### The problem

Steps to reproduce: 

```
curl -X GET -H "Content-Type: application/json" 'http://192.168.86.44:8080/v1/groups/+15853710007' > out.json && python -m json.tool out.json | less
```

And getting this response: 

```
{
    "error": "json: cannot unmarshal object into Go struct field SignalCliGroupEntry.pendingMembers of type string"
}
```

I would expect to get the regular JSON output with the content of the groups. 

### Are you using the latest released version?

- [X] Yes

### Have you read the troubleshooting page?

- [X] Yes

### What type of installation are you running?

signal-cli-rest-api Docker Container

### In which mode are you using the docker container?

Normal Mode

### What's the architecture of your host system?

x86-64

**Comment by kocsenc** [comment]
I got output of the logs in case it's helpful

```
+ set -e
+ [ -z /home/.local/share/signal-cli ]
+ chown 1000:1000 -R /home/.local/share/signal-cli
+ cat
+ cap_prefix=-cap_
+ cat /proc/sys/kernel/cap_last_cap
+ seq -s ,-cap_ 0 37
+ caps=-cap_0,-cap_1,-cap_2,-cap_3,-cap_4,-cap_5,-cap_6,-cap_7,-cap_8,-cap_9,-cap_10,-cap_11,-cap_12,-cap_13,-cap_14,-cap_15,-cap_16,-cap_17,-cap_18,-cap_19,-cap_20,-cap_21,-cap_22,-cap_23,-cap_24,-cap_25,-cap_26,-cap_27,-cap_28,-cap_29,-cap_30,-cap_31,-cap_32,-cap_33,-cap_34,-cap_35,-cap_36,-cap_37
+ [  = json-rpc ]
+ exec setpriv --reuid=1000 --regid=1000 --init-groups --inh-caps=-cap_0,-cap_1,-cap_2,-cap_3,-cap_4,-cap_5,-cap_6,-cap_7,-cap_8,-cap_9,-cap_10,-cap_11,-cap_12,-cap_13,-cap_14,-cap_15,-cap_16,-cap_17,-cap_18,-cap_19,-cap_20,-cap_21,-cap_22,-cap_23,-cap_24,-cap_25,-cap_26,-cap_27,-cap_28,-cap_29,-cap_30,-cap_31,-cap_32,-cap_33,-cap_34,-cap_35,-cap_36,-cap_37 signal-cli-rest-api -signal-cli-config=/home/.local/share/signal-cli
time="2021-12-06T05:39:09Z" level=info msg="Started Signal Messenger REST API"
[GIN] 2021/12/06 - 05:46:51 | 400 |  4.318060018s |       127.0.0.1 | POST     "/v1/register/+15853718837"
[GIN] 2021/12/06 - 05:48:45 | 201 |  6.072697648s |       127.0.0.1 | POST     "/v1/register/+15853718837"
[GIN] 2021/12/06 - 05:50:14 | 201 |  5.870740247s |       127.0.0.1 | POST     "/v1/register/+15853718837/verify/966015"
[GIN] 2021/12/06 - 05:50:28 | 201 | 13.527720816s |       127.0.0.1 | POST     "/v1/groups/+15853718837"
[GIN] 2021/12/06 - 05:50:34 | 400 |  5.934351819s |       127.0.0.1 | GET      "/v1/groups/+15853718837"
[GIN] 2021/12/06 - 05:51:46 | 400 |  6.506541505s |       127.0.0.1 | GET      "/v1/groups/+15853718837"
[GIN] 2021/12/06 - 05:52:19 | 400 |  5.922249454s |       127.0.0.1 | GET      "/v1/groups/+15853718837"
[GIN] 2021/12/06 - 05:52:37 | 400 |  5.183688103s |       127.0.0.1 | GET      "/v1/groups/+15853718837"
[GIN] 2021/12/06 - 05:54:16 | 400 |  8.730499754s |   192.168.86.56 | GET      "/v1/groups/+15853718837"
[GIN] 2021/12/06 - 05:54:19 | 404 |         599ns |   192.168.86.56 | GET      "/"
[GIN] 2021/12/06 - 05:54:19 | 404 |         540ns |   192.168.86.56 | GET      "/favicon.ico"
[GIN] 2021/12/06 - 05:55:40 | 400 |  3.782654034s |       127.0.0.1 | GET      "/v1/groups/+15853718837"
[GIN] 2021/12/06 - 05:57:15 | 400 |  5.437351933s |       127.0.0.1 | GET      "/v1/groups/+15853718837"
```

**Comment by bbernhard** [comment]
I've seen this before, but wasn't able to reproduce the issue here. Could you please enable debug logging and post the output from `signal-cli` here (see  https://github.com/bbernhard/signal-cli-rest-api/blob/master/doc/DEBUG.md for details). Please don't forget to obfuscate the `signal-cli` output!

**Comment by kocsenc** [comment]
Debug output when running the `signal-cli` directly on container: 


```
signal-api@teslamate:/$  signal-cli --config /home/.local/share/signal-cli --output json -u +<REDACTED> listGroups -d
INFO ManagerImpl - The Signal protocol expects that incoming messages are regularly received.
[{"id":"<REDACTED>/<REDACTED>=","name":"<REDACTED>","description":"","isMember":true,"isBlocked":false,"messageExpirationTime":0,"members":[{"number":"+<REDACTED>","uuid":"dd9bc2be-c706-45c5-85fb-<REDACTED>"}],"pendingMembers":[{"number":"+<REDACTED>","uuid":"e8769d7d-aa3b-4020-b724-<REDACTED>"},{"number":"+<REDACTED>","uuid":"94de218a-42a8-4205-874b-<REDACTED>"}],"requestingMembers":[],"admins":[{"number":"+<REDACTED>","uuid":"dd9bc2be-c706-<REDACTED>-85fb-<REDACTED>"}],"permissionAddMember":"EVERY_MEMBER","permissionEditDetails":"EVERY_MEMBER","permissionSendMessage":"EVERY_MEMBER","groupInviteLink":null}]
```

**Comment by bbernhard** [comment]
Please give this docker test image `bbernhard/signal-cli-rest-api:0.83-signal-cli-groups-fix-dev` a try. It should (hopefully) fix your issue.

**Comment by bbernhard** [comment]
Should be fixed with Version 0.54. In case the issue still persists, please re-open.
